frontend/src/scenes/events/Events.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { kea, useActions, useValues } from 'kea'
+
+import { hot } from 'react-hot-loader/root'
+import { PageHeader } from 'lib/components/PageHeader'
+import { Tabs } from 'antd'
+import { ActionsTable } from 'scenes/actions/ActionsTable'
+import { EventsTable } from './EventsTable'
+import { EventsVolumeTable } from './EventsVolumeTable'
+import { PropertiesVolumeTable } from './PropertiesVolumeTable'
+
+const eventsLogic = kea({
+    actions: {
+        setTab: (tab: string) => ({ tab }),
+    },
+    reducers: {
+        tab: [
+            'live',
+            {
+                setTab: (_, { tab }) => tab,
+            },
+        ],
+    },
+    actionToUrl: ({ values }) => ({
+        setTab: () => '/events' + (values.tab === 'live' ? '' : '/' + values.tab),
+    }),
+    urlToAction: ({ actions, values }) => ({
+        '/events(/:tab)': ({ tab }: Record<string, string>) => {
+            if (tab !== values.tab && tab) actions.setTab(tab)
+        },
+    }),
+})
+
+export const ManageEvents = hot(_ManageEvents)
+function _ManageEvents({}): JSX.Element {
+    const { tab } = useValues(eventsLogic)
+    const { setTab } = useActions(eventsLogic)
+
+    return (
+        <div className="manage-events" data-attr="manage-events-table">
+            <PageHeader title="Manage Events" />
+            <Tabs tabPosition="top" animated={false} activeKey={tab} onTabClick={setTab}>
+                <Tabs.TabPane tab="Live Events" key="live">
+                    <i>See all events that are being sent to this team in real time.</i>
```

Comment:
"sent to this **project**" (same for properties table), "near real-time" maybe?

Also following new design guidelines, this should not be italicized.